### PR TITLE
shuffle choices

### PR DESCRIPTION
**Pull request by esa-coorp**
Bump:
```
    "@coorpacademy/player-services": "5.1.14",
    "@coorpacademy/player-store": "4.1.22",
    "@coorpacademy/progression-engine": "11.1.9",
```
**shuffleChoices: false** for test only

### Testing strategy

- [ ] I ensured my code is tested: 
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/redux/actions/progressions/create-chapter-progression.js [hunk 1]
```diff
@@ -14,5 +15,9 @@ export const createChapterProgression = (chapter: Chapter, engineVersion?: strin
   const content: GenericContent = {type: CONTENT_TYPE.CHAPTER, ref};
   const engineConfig: EngineConfig = {version: engineVersion || 'latest'};
 
+  if (__TEST__) {
```
Comment (alanlanglois) [inline src/redux/actions/progressions/create-chapter-progression.js:18]: Wouldn't be possible to add it in the definition of EngineConfig as a paramater?

File: src/redux/actions/progressions/create-level-progression.js [hunk 2]
```diff
@@ -18,5 +19,9 @@ export const createLevelProgression = (level: Level, engineVersion?: string) =>
     livesDisabled: level.infiniteLives
   };
 
+  if (__TEST__) {
```
Comment (alanlanglois) [inline src/redux/actions/progressions/create-level-progression.js:22]: Same as above

**Comment by alanlanglois** [comment]
Autre soucis sur Android qui peut éventuellement expliquer le pb de build probablement lié au bump de nova-icon aussi : 
image

File: src/redux/actions/progressions/create-chapter-progression.js (hunk 1 above)
Comment (esa-coorp) [reply src/redux/actions/progressions/create-chapter-progression.js:18]: It's the case. I overload the config for the tests

Comment (alanlanglois) [reply src/redux/actions/progressions/create-chapter-progression.js:18]: I meant like this: `const engineConfig: EngineConfig = {version: engineVersion || 'latest', shuffleChoices: (__TEST__) ? false : true};`
But I'm ok with what you've done, I was just wondering if the way I suggest is correct.

File: src/redux/actions/progressions/create-level-progression.test.js [hunk 3]
```diff
@@ -14,7 +14,66 @@ describe('createLevelProgression', () => {
...
+    });
+
+    // $FlowFixMe
+    const actual = await createLevelProgression(level, version);
+
+    expect(playerStore.createProgression).toHaveBeenCalledTimes(1);
+    expect(actual).toEqual({type: '@@mock/CREATE_PROGRESSION', payload: {_id: '__ID__'}});
+  });
+
+  it('should create level progression with specific engine version in test', async () => {
```
Comment (Fausto95) [inline src/redux/actions/progressions/create-level-progression.test.js:44]: Duplicated, please remove..

File: src/redux/actions/progressions/create-level-progression.test.js [hunk 4]
```diff
@@ -14,7 +14,66 @@ describe('createLevelProgression', () => {
...
+    });
+
+    // $FlowFixMe
+    const actual = await createLevelProgression(level);
+
+    expect(playerStore.createProgression).toHaveBeenCalledTimes(1);
+    expect(actual).toEqual({type: '@@mock/CREATE_PROGRESSION', payload: {_id: '__ID__'}});
+  });
+
+  it('should create level progression with specific engine version in production', async () => {
```
Comment (Fausto95) [inline src/redux/actions/progressions/create-level-progression.test.js:73]: Also duplicated

**Comment by Fausto95** [comment]
I will merge this after #542

**Pull request merged**